### PR TITLE
Enhance SpaceBetweenAlphabeticalWordValidator

### PR DESCRIPTION
**Pull request by midnightSuyama**
Add skip character of SpaceBetweenAlphabeticalWordValidator.

## New properties

* skip_before_chars
* skip_after_chars

### Document file

```
iPhone は iPod・携帯電話・インターネットの融合。
```

### Configuration file

```
        <validator name="SpaceBetweenAlphabeticalWord">
            <property name="skip_before_chars" value="。、・（「『"/>
            <property name="skip_after_chars" value="。、・）」』"/>
        </validator>
```

### CODE REVIEW

**Comment by takahi-i** [comment]
Very sorry for the late response.. and thank you very much for the valuable changes @midnightSuyama!

File: redpen-core/src/main/java/cc/redpen/validator/sentence/SpaceBetweenAlphabeticalWordValidator.java [hunk 1]
```diff
@@ -40,7 +40,9 @@
     private final Pattern pat = Pattern.compile(shard + "\\s+(" + word + ")\\s+" + shard);
 
     public SpaceBetweenAlphabeticalWordValidator() {
-        super("forbidden", false); // Spaces are enforced (false) or forbidden (true)
+        super("forbidden", false, // Spaces are enforced (false) or forbidden (true)
+              "skip_before_chars", "",
```
Comment (takahi-i) [inline redpen-core/src/main/java/cc/redpen/validator/sentence/SpaceBetweenAlphabeticalWordValidator.java]: I would like to shorten the configuration names.  If you are ok, could you please rename ```skip_before_chars``` into ```skip_before``` and  ```skip_after_chars``` to ```skip_after```?

**Comment by midnightSuyama** [comment]
@takahi-i All right, renamed.

Thanks :)

**Comment by takahi-i** [comment]
LGTM

**Comment by takahi-i** [comment]
Thanks a lot @midnightSuyama! Merged.

**Comment by takahi-i** [comment]
Ref issue for this feature: https://github.com/redpen-cc/redpen-doc/issues/66

**Comment by takahi-i** [comment]
Added the usage: https://github.com/redpen-cc/redpen-doc/pull/68

**Pull request merged**